sandbox/src/test/java/com/badoo/ribs/sandbox/rib/switcher/SwitcherInteractorTest.kt

```diff
@@ -21,7 +21,10 @@ import com.nhaarman.mockitokotlin2.verify
 import io.reactivex.Observable
 import org.junit.Before
 import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
 
+@RunWith(RobolectricTestRunner::class)
```

Comment:
When adding the `deps.compose.ui` dependency, the unit tests fail with the following error:
https://github.com/badoo/RIBs/runs/1981140159?check_suite_focus=true

`java.lang.RuntimeException: Method getMainLooper in android.os.Looper not mocked.`

Reply:
Running this with Robolectric appears to resolve the issue. I presume the dependency is causing mischief